module-4/backend/week_four.md

```diff
@@ -1,19 +1,57 @@
 ## Week Four - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+1. What's your favorite project management tool?
 
-Note: When you're done, submit a PR. 
+Waffle.io
 
-1. What's your favorite project management tool?
 2. Why do we create staging environments?
+
+To ensure deployment is pain free and compatible across many machines. 
+
 3. What are the characteristics of a good README (in your opinion)?
+
+- Clear requirements for versions
+- Clear setup instructions
+- Maybe some screen shots of the in progress project
+- A description of the app or repo
+- Possible contribution instructions if it's open source
+
 4. What's one main improvement you're going to make to your code regarding accessibility issues?
+
+Better READMEs on all my repo's. Reactive styling so that content can be viewed on any device. Adding ALT's to all images for screen readers. 
+
 5. What are some basic security concerns to be aware of when building applications?
+
+- Broken Authentications
+- Injection
+- Insufficient logging and monitoring.
+
+
 6. Why is continuous integration helpful/important?
+
+Allows for feedback while in development which generally leads to a better product that fits the customers needs more closely. 
```

Comment:
it allows you to easily run your entire test suite to ensure your changes haven't broken another part of the application.